spec.html

```diff
@@ -13768,6 +13769,7 @@ <h1>Runtime Semantics: Evaluation</h1>
         1. Let _rref_ be the result of evaluating |RelationalExpression|.
         1. Let _rval_ be ? GetValue(_rref_).
         1. Let _r_ be the result of performing Strict Equality Comparison _rval_ === _lval_.
+        1. Assert: Type(_r_) is Boolean.
```

Comment:
Is `r` technically a completion record here? Should we be explicit and unwrap it?